runtime-testsuite/OLD-descriptors/ParserErrorsDescriptors.java

```diff
@@ -639,7 +639,7 @@
 
 		@Override
 		public boolean ignore(String targetName) {
-			return !"Java".equals(targetName) && !"Swift".equals(targetName) && !"Dart".equals(targetName);
+			return !"".equals(targetName) && !"Swift".equals(targetName) && !"Dart".equals(targetName);
```

Comment:
Why `Java` is removed here and below?

Reply:
hmm...good question. must be a typo or something weird when I was editing. hmm..yeah, then i saw that as always ignoring. It was failing java. let me add descr back and see.

Reply:
ok, descriptor seems to work. I added

```
[type]
Parser

[grammar]
grammar T;

member : 'a';
body : member*;
file : body EOF;
B : 'b';

[start]
file

[input]
baa

[output]

[errors]
"""line 1:0 mismatched input 'b' expecting {<EOF>, 'a'}
"""

[skip]
Cpp
CSharp
Go
Node
PHP
Python2
Python3
```